File: MPI/mpicheck/tests/functions/4.0/MPI_Alltoallw_init.f

        
        program main
        include 'mpif.h'
        
        TYPE(INTEGER) var_0(10)
       INTEGER var_1(10)
       INTEGER var_2(10)
       INTEGER var_3(10)
       TYPE(INTEGER) var_4(10)
       INTEGER var_5(10)
       INTEGER var_6(10)
       INTEGER var_7(10)
       INTEGER var_8
       INTEGER var_9
       INTEGER var_10
       INTEGER var_11
        call mpi_alltoallw_init(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9, var_10, var_11)
       call pmpi_alltoallw_init(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9, var_10, var_11)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Alltoallw_init.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    const int var_1[2];
    const int var_2[2];
    const MPI_Datatype var_3[2];
    void *var_4;
    const int var_5[2];
    const int var_6[2];
    const MPI_Datatype var_7[2];
    MPI_Comm var_8;
    MPI_Info var_9;
    MPI_Request *var_10;
    int ret;
    /* calls */
    ret = MPI_Alltoallw_init(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, var_9, var_10);
    ret = PMPI_Alltoallw_init(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, var_9, var_10);
    return 0;
}
